topydo/ui/TodoListWidget.py

```diff
@@ -240,6 +261,8 @@ def resolve_action(self, p_action_str, p_size=None):
                 self._execute_on_selected(cmd, execute_signal)
             else:
                 urwid.emit_signal(self, execute_signal, cmd)
+
+            urwid.emit_signal(self, 'save_cmd', cmd, execute_signal)
         else:
```

Comment:
That makes sense, but it would require significant changes. Right now `UIApplication._execute_handler` gets complete todo command (with potential todo id already supplied) or uncomplete (with `{}` placeholder in place of todo id). Depending on that it is decided to use `pending_todos` or not.

If we want to remove the signal above we have to:
1. force `TodoListWidget._execute_on_selected` to **always** send id of focused todo item.
2. Decide upon `pending_todos` vs sent todo id based on value of `pending_todos`
3. Change meaning (and type) of `p_command` argument in `UIApplication._execute_handler` to tuple containing command string (with potential `{}` placeholder) and potential todo id.
4. force all clients of `UIApplication._execute_handler` (for instance command-line widget) to send tuples instead of strings.

Points 3 and 4 can also be resolved by adding supplemental argument for `UIApplication._execute_handler` but that would of course require to adjust client calls.